app/models/concerns/iiif_print/iiif_print_behavior.rb

```diff
@@ -4,5 +4,14 @@ module IiifPrintBehavior
     def split_pdf
       true
     end
+
+    def ancestor_ids(o)
```

Comment:
I commented `#ancestor_ids` out of `FileSetIndexer` and `ChildIndexer` and i was able to get this in console:

```rb
Hyrax::FileSetIndexer.new(FileSet.new).method(:ancestor_ids).source_location
#=> ["/app/samvera/hyrax-webapp/vendor/engines/iiif_print/app/models/concerns/iiif_print/iiif_print_behavior.rb", 8]

GenericWorkIndexer.new(GenericWork.new).method(:ancestor_ids).source_location
#=> ["/app/samvera/hyrax-webapp/vendor/engines/iiif_print/app/models/concerns/iiif_print/iiif_print_behavior.rb", 8]
```

Reply:
I fixed this and it's working now. Thanks for checking.